ledger/src/bigtable_upload.rs

```diff
@@ -223,6 +257,35 @@ pub async fn upload_confirmed_blocks(
 
     measure.stop();
     info!("{}", measure);
+
+    let blockstore_results: Vec<std::thread::Result<BlockstoreLoadStats>> =
+        loader_threads.into_iter().map(|t| t.join()).collect();
```

Comment:
I think the `collect()` here is superfluous?  Should be able to just update the stats directly from the iterator

Reply:
fixed